scripts/ci/run-vagrant-tests.sh

```diff
@@ -23,8 +23,20 @@ trap vagrant_cleanup EXIT
 export ANALYZER_COUNT=2
 export AGENT_COUNT=1
 
-for mode in dev binary package container
+for mode in binary package container
 do
+  if [ "$DEVMODE" == "true" ]; then
+    case $mode in
+      dev)
+        make install ;;
```

Comment:
need make -C  ${GOPATH}/src/github.com/skydive-project/skydive for all make call